README.rst

```diff
@@ -1,8 +1,13 @@
 Vulture - Find dead code
 ========================
 
-.. image:: https://travis-ci.org/jendrikseipp/vulture.svg?branch=master
+.. image:: https://travis-ci.org/jendrikseipp/vulture.svg?branch=master&label=Linux%20build
```

Comment:
It seems the label parameter is ignored. Is that so? If yes, I think it should be removed.

Reply:
Oops, I meant to also serve this from https://img.shields.io/travis/jendrikseipp/vulture.svg?branch=master&label=Linux%20build so the parameter would be used. 

Which would you prefer: remove the parameter or serve from same place?

Reply:
I think it would be best if we could avoid using another service (i.e., shields.io). Can we use https://www.appveyor.com/docs/status-badges/ directly? The titles are not so important.

Reply:
I've removed the Travis parameter.

I can't see the AppVeyor badge URL, they hide it away:

> You can see a badge image URL and sample markup on the Badges tab of project settings. Badge URLs for both private and public projects contain a security token.

Please can you paste the URL here or update the PR directly?